qemu/tests/cfg/blockdev_commit_reboot.cfg

```diff
@@ -0,0 +1,34 @@
+- blockdev_commit_reboot:
+    type = blockdev_commit_reboot
+    virt_test_type = qemu
+    only Linux
+    start_vm = yes
+    kill_vm = yes
+    storage_pools = default
+    storage_type_default = "directory"
+    storage_pool = default
+    image_size_data = 500M
+    image_name_data = data
+    snapshot_tags = sn1 sn2 sn3 sn4
+
+    image_size_sn1 = 20G
+    image_name_sn1 = sn1
+    image_format_sn1 = qcow2
+
+    image_name_sn2 = sn2
+    image_size_sn2 = 20G
+    image_format_sn2 = qcow2
+
+    image_name_sn3 = sn3
+    image_size_sn3 = 20G
+    image_format_sn3 = qcow2
+
+    image_name_sn4 = sn4
+    image_size_sn4 = 20G
+    image_format_sn4 = qcow2
+
+    device_tag = "image1"
+    format = qcow2
+    rebase_mode = unsafe
+    Host_RHEL.m8:
+        qemu_force_use_drive_expression = no
```

Comment:
you can see that there are so much duplicated informat in cfg files, such as sn1/2/3/4, is it possible to define one cfg file like blockdev_commit.cfg, then put the reboot/commit as its variants? At that time, you might get a clean cfg.

Reply:
Yes, but was suggested one case, one cfg is better.